tests/summarizer/test_openai_summarizer.py

```
@@ -0,0 +1,315 @@
+            title,
+            description,
+            output_file,
+            topics
+        )
+
+        # Verify file was created
+        assert os.path.exists(output_file)
+        assert result == output_file
+
+        # Verify content
+        with open(output_file, 'r', encoding='utf-8') as f:
```

Ruff (0.14.5)

Unnecessary mode argument

Remove mode argument

(UP015)